Makefile

```diff
@@ -198,6 +199,10 @@ cmd/grafana-agent-crow/grafana-agent-crow: cmd/grafana-agent-crow/main.go
 	$(ALL_CGO_BUILD_FLAGS) ; $(seego) build $(CGO_FLAGS) -o $@ ./$(@D)
 	$(NETGO_CHECK)
 
+cmd/agent-smoke/agent-smoke: seego cmd/agent-smoke/main.go
```

Comment:
nit: we should start prefixing binaries `grafana-agent-` instead of just `agent-` (see #582)